### PR TITLE
shorten format of timeouts shown in error messages

### PR DESCRIPTION
**Pull request by asolntsev**
"100 ms." -> "100ms"

I've removed space and dot. It's readable, compact and resistant to punctuation edge cases.

### CODE REVIEW

File: modules/grid/src/test/java/integration/DownloadFileFromGridToFolderTest.java [hunk 1]
```diff
@@ -91,25 +91,26 @@ void failFastOnNotMatchingFile() {
...
       .isInstanceOf(FileNotDownloadedError.class)
-      .hasMessageStartingWith("Failed to download file with name \"foo.bar\" in 120000 ms")
-      .hasMessageFindingMatch("haven't been modified for 50\\d{2}");
+      .hasMessageStartingWith("Failed to download file with name \"foo.bar\" in 10s")
+      .hasMessageFindingMatch("haven't been modified for 1\\.\\d{3}s")
+      .hasMessageContaining("incrementTimeout: 1.002s");
 
     long end = currentTimeMillis();
-    assertThat(end - start).isLessThan(120_000L);
+    assertThat(end - start).as("Less than timeout (but greater than increment timeout)").isBetween(1002L, 5_000L);
```
Comment (rosolko) [inline modules/grid/src/test/java/integration/DownloadFileFromGridToFolderTest.java]: Average download method execution time is something around 6700ms
